packages/ipfs-http-server/tsconfig-types.aegir.json

```diff
@@ -0,0 +1 @@
+{"extends":"../../tsconfig.json","compilerOptions":{"outDir":"dist","noEmit":false,"emitDeclarationOnly":true},"include":["src","package.json"],"references":[{"path":"../ipfs-core-utils"},{"path":"../ipfs-core"},{"path":"../ipfs-http-gateway"}],"exclude":["test"]}
```

Comment:
I think this file has been added by accident.

Reply:
It's added to the directory tree and also deleted by aegir during the build, so this can get left around if you `ctrl+c` before it's finished.  Really it should go in the `.gitignore` file.